src/xcb_ffi/raw_ffi.rs

```diff
@@ -1,11 +1,20 @@
-#[cfg(not(test))]
-use std::io::IoSlice;
 use std::ptr::NonNull;
 
-#[cfg(not(test))]
+#[cfg(not(all(test, unix)))]
 use libc::c_void;
+#[cfg(unix)]
+pub(crate) use libc::iovec;
 use libc::{c_char, c_int, c_uint};
 
+// As defined in xcb_windefs.h
+#[cfg(not(unix))]
+#[derive(Copy, Clone, Debug)]
```

Comment:
Why `Copy` and `Clone`? `std::io::IoSlice` only has `Debug` and `Deref`.

Reply:
But `iovec` from libc [has `Copy` and `Clone`](https://docs.rs/libc/0.2.67/libc/struct.iovec.html) (but not `Debug`).

There is a [PR](https://github.com/rust-lang/rust/issues/69395) to implemente `Copy` and `Clone` for `std::io::IoSlice`.

Anyways we are not using them and the definition is private, so we could remove them.